hail/python/hail/experimental/vcf_combiner.py

```diff
@@ -39,7 +39,10 @@ def transform_one(mt: MatrixTable) -> MatrixTable:
 
 
 def merge_alleles(alleles) -> ArrayExpression:
-    return hl.array(hl.set(hl.flatten(alleles)))
+    tmp = hl.array(hl.set(hl.flatten(alleles)))
+    non_ref = hl.filter(lambda a: a.alt == '<NON_REF>', tmp)
```

Comment:
Is there always a non ref allele?

Reply:
also how does the ref work here?

Reply:
This function takes an `array<array<struct{ref:str, alt:str}>>`. I suppose that there is always a `<NON_REF>` allele, but I never check that it exists.

The intention is for this function to be called on the output of the `min_rep` function in `combine_gvcfs` below.